packages/app-store/alby/lib/parseInvoice.ts

```diff
@@ -9,7 +9,13 @@ export default function parseInvoice(
     "svix-signature": string;
   },
   webhookEndpointSecret: string
-): Invoice {
-  const wh = new Webhook(webhookEndpointSecret);
-  return wh.verify(body, headers) as Invoice;
+): Invoice | null {
+  try {
+    const wh = new Webhook(webhookEndpointSecret);
+    return wh.verify(body, headers) as Invoice;
+  } catch (err) {
+    // Looks like alby might sent multiple webhooks for the same invoice but it should only work once
```

Comment:
This will happen if you disconnect and reconnect the Alby app (the old webhook is not removed and the secret won't match). I will add a TODO for this.

Reply:
Yeah, removing old webhook on disconnect.

Reply:
Currently there are some issues on our side here. We can store an OAuth refresh/access token for the app but it'll expire after 30 days and we can only refresh it if there's a booking within that period. If there isn't, we won't be able to call the endpoint to remove the webhook because we won't have a valid token.

For now it seems best to just leave the webhook there and not deal with that additional complexity in the initial version (if the user removes or reinstalls the app, the old webhook calls won't pass the endpoint secret checks and just log an error, but not break anything).